packages/components/pie-input/src/index.ts

```diff
@@ -20,6 +20,7 @@ import 'element-internals-polyfill';
 export * from './defs';
 
 const componentSelector = 'pie-input';
+const assistiveTextValue = 'assistive-text';
```

Comment:
```suggestion
const assistiveTextId = 'assistive-text';
```

Reply:
You ok if i change it to `assistiveTextIdValue` as it represents the value rather than the ID. I had originally thought about using this name but went with value instead.